src/__testing__/App.test.js

```diff
@@ -0,0 +1,22 @@
+import { render } from '@testing-library/react';
+import { BrowserRouter } from 'react-router-dom';
+import { Provider } from 'react-redux';
+import App from '../App';
+import Store from '../Redux/ConfigureStore';
+
+const MockApp = () => (
```

Comment:
- [ ] One of your tests is failing. It is in the `src/App.test.js` file. Kindly fix that.

  ![ddudu](https://user-images.githubusercontent.com/81468675/206138102-cb9005ad-8f12-4a6d-9b92-5c519774509d.jpg)